config/DELFI.BB-flex/trips.txt

```diff
@@ -0,0 +1,3 @@
+route_id,service_id,trip_id,trip_headsign,trip_short_name,block_id,shape_id,wheelchair_accessible,bikes_allowed
```

Comment:
The routes.txt names additional routes, which not yet have trips or stop_times. I assume, you will add them in a later PR?

Reply:
Oops, I accidentally committed that, even though it was just a draft. Will remove it.